presto-docs/src/main/sphinx/release/release-0.156.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.156
+=============
+
+General Changes
+---------------
+
+* Add support for quantified comparison predicates: ``ALL``, ``ANY`` and ``SOME``.
+* Change default task concurrency to 16.
+* Add support for ``ARRAY`` and ``ROW`` in :func:`checksum` aggregation.
```

Comment:
Those types were already supported. More precisely, it's support for arrays and rows containing nulls.